src/catkin_pkg/cli/generate_changelog.py

```diff
@@ -40,16 +40,20 @@ def prompt_continue(msg, default):
 
 def main(sysargs=None):
     parser = argparse.ArgumentParser(description='Generate a REP-0132 %s' % CHANGELOG_FILENAME)
+    group_merge = parser.add_mutually_exclusive_group()
     parser.add_argument(
         '-a', '--all', action='store_true', default=False,
         help='Generate changelog for all versions instead of only the forthcoming one (only supported when no changelog file exists yet)')
+    group_merge.add_argument(
+        '--only-merges', action='store_true', default=False,
+        help='Skip all commits but merge commits to the changelog')
```

Comment:
Please update help text to:

> Only add merge commits to the changelog

(I would have done the edit myself but it seems that flag has been disabled for this PR.)

Reply:
That makes sense, updated.